package.json

```diff
@@ -27,10 +27,13 @@
     "prebuild": "npm run doc-listing-inject",
     "build": "node scripts/build.js",
     "postbuild": "rm -rf examples/node_modules && mkdir examples/node_modules && cp -r node_modules/@webcomponents node_modules/requirejs examples/node_modules/",
-    "clean": "rm -rf node_modules package-lock.json elements/*/{node_modules,package-lock.json,_temp} test/vrt-snapshots",
+    "clean": "rm -rf node_modules elements/*/{node_modules,package-lock.json}",
+    "clean:components": "rm -rf elements/*/{dist,_temp}",
+    "clean:e2e": "rm -rf test/vrt-snapshots",
     "bootstrap": "lerna bootstrap --hoist",
     "deploy-docs": "bash scripts/deploy-docs.sh",
     "release": "bash scripts/release.sh",
+    "preinstall": "npm run clean",
```

Comment:
How much does this increase the time it takes to npm install? I guess my only concern would be making every install a clean install when you might only need a dependency update or something.

Another question is if you were to npm install a single module would it clean everything (and not install all the other dependencies)?

I think I'd prefer to keep this a separate script that you could use as needed rather than forcing it every time. 🤔

Reply:
Yeah, I think you're right.  The ci command is actually failing on this and I'm debugging it.  I think I'll just remove it.